namingsystems/NamingSystem-England-ERShealthcareserice.xml

```diff
@@ -0,0 +1,25 @@
+<NamingSystem xmlns="http://hl7.org/fhir">
+    <id value="NamingSystem-England-ERShealthcareserice"/>
```

Comment:
NamingSystem-England-ERShealthcareserice-> England-ERSHealthcareService [to be discussed]